pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -114,9 +114,9 @@ func (rc *reconciler) reconciliationLoopFunc() func() {
 						glog.Infof("Started DetachVolume for volume %q from node %q", attachedVolume.VolumeName, attachedVolume.NodeName)
 					}
 					if err != nil &&
-						!goroutinemap.IsAlreadyExists(err) &&
-						!goroutinemap.IsExponentialBackoff(err) {
-						// Ignore goroutinemap.IsAlreadyExists && goroutinemap.IsExponentialBackoff errors, they are expected.
+						!common.IsAlreadyExists(err) &&
```

Comment:
@saad-ali @smarterclayton Even if the implementation is in common (which I am not sure makes sense) I think we can wrap it up in the goRoutine func